pkg/api/customization/clusterregistrationtokens/formatter.go

```diff
@@ -31,9 +38,14 @@ func Formatter(request *types.APIContext, resource *types.RawResource) {
 			image.Resolve(settings.AgentImage.Get()),
 			getRootURL(request),
 			token,
-			ca)
+			caNonWindows)
 		resource.Values["token"] = token
 		resource.Values["manifestUrl"] = url
+		resource.Values["windowsNodeCommand"] = fmt.Sprintf(windowsNodeCommandFormat,
```

Comment:
if this is now a field we are returning, it needs to be defined in rancher/types

Reply:
@cjellick , https://github.com/rancher/types/pull/551